elytron/src/main/java/org/wildfly/extension/elytron/CertificateChainAttributeDefinitions.java

```diff
@@ -88,9 +89,15 @@ class CertificateChainAttributeDefinitions {
 
     // TODO - Consider adding some of the more detailed fields from X509Certificate
 
+    // derivative attributes
+    private static final SimpleAttributeDefinition VALIDITY = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.VALIDITY, ModelType.STRING)
```

Comment:
I think we can also specify the units in the definition instead of in the text, I would propose we use units that are more user friendly.

I don't think it is reasonable to check more frequently than once per second, even that sounds too frequent and maybe the granularity should be in minutes.

Reply:
Oh, honestly I just forgot thats possible...

Reply:
Ach, actually its by design, its string representation not a numeric value(TTL).

Reply:
Thank you, I think my comment is on the wrong attribute - I see this one is the info from the certifcate.